ByBit.Net/Clients/SpotApi/v3/BybitClientSpotApiTradingV3.cs

```diff
@@ -219,7 +219,8 @@ public async Task<WebCallResult<IEnumerable<BybitBorrowRecord>>> GetBorrowRecord
             parameters.AddOptionalParameter("limit", limit);
             parameters.AddOptionalParameter("recvWindow", receiveWindow?.ToString(CultureInfo.InvariantCulture) ?? _baseClient.ClientOptions.ReceiveWindow.TotalMilliseconds.ToString(CultureInfo.InvariantCulture));
 
-            return await _baseClient.SendRequestAsync<IEnumerable<BybitBorrowRecord>>(_baseClient.GetUrl("spot/v3/private/cross-margin-orders"), HttpMethod.Get, ct, parameters, true).ConfigureAwait(false);
+            var result = await _baseClient.SendRequestAsync<BybitBorrowRecordWrapper>(_baseClient.GetUrl("spot/v3/private/cross-margin-orders"), HttpMethod.Get, ct, parameters, true).ConfigureAwait(false);
+            return result.As(result.Data.Borrows);
```

Comment:
This has the same issue as the earlier PR, result.Data is null when the call fails

Reply:
I do like your idea with BybitList. Converted to it.